Sources/SymbolKit/UnifiedSymbolGraph/UnifiedSymbol.swift

```diff
@@ -98,3 +98,159 @@ extension UnifiedSymbolGraph {
         }
     }
 }
+
+extension UnifiedSymbolGraph.Symbol: Encodable {
```

Comment:
I really like how this is isolated into an extension like this. Maybe it's worth also splitting into a separate file (`UnifiedSymbol+Encodable.swift`)? What do you think?

Reply:
Should the same be done for `UnifiedSymbolGraph` itself, since its `Encodable` conformance is also in its own extension block?

Reply:
I think that might be nice. Since these models are non-trivial it feels like it's a little easier to parse these different components if we keep them more isolated.

Reply:
I can get behind that; since i had to jump through some hoops to write up the format the way i wanted, it's worth keeping it by itself so it doesn't inflate the base file itself. I'll do that.